packages/docusaurus/src/client/App.tsx

```diff
@@ -17,6 +18,19 @@ import PendingNavigation from './PendingNavigation';
 
 import './client-lifecycles-dispatcher';
 
+function CSSLoadingWarningMessage({isClient}) {
+  const location = useLocation();
+  return (
+    <div style={{border: 'solid red thick', padding: '16px'}}>
+      <span>
+        You site CSS did not load properly. Your baseUrl setting is probably
+        bad.{' '}
+      </span>
+      {isClient && <span>Maybe try baseUrl = {location.pathname}</span>}
```

Comment:
Here it's not totally correct. 

If we are on:
- `/myAppBaseUrl/`, we should recommend the user to try `baseUrl: '/myAppBaseUrl/'`
- `/myAppBaseUrl/docs/myDoc`, we should recommend the user to try `baseUrl: '/myAppBaseUrl/'`

What you currently have is, if you are on:
- `/myAppBaseUrl/`, you recommend the user to try `baseUrl: '/myAppBaseUrl/'`
- `/myAppBaseUrl/docs/myDoc`, you recommend the user to try `baseUrl: '/myAppBaseUrl/docs/myDoc'` **=> not a good recommendation**

Not sure what's the best way to implement this, but my initial suggestion was rather to only display this message on the homepage, to avoid increasing all the site's page size with this additional, warning, and as it's the URL users will tend to browse first when trying their new deployment. 

In any case we should only recommend valid baseUrl settings to the user, or he'll be more confused than ever.

Reply:
So should I remove the suggestion for now?